src/decide.ts

```diff
@@ -85,6 +85,20 @@ export function compareFileTrees(
 	);
 }
 
+export function compareFileTreesIgnoringFolders(
+	candidate: Metafile,
+	searchee: Searchee
+): boolean {
+	const cmp = (elOfA: File, elOfB: File) => {
+		const lengthsAreEqual = elOfB.length === elOfA.length;
+		const pathsAreEqual = elOfB.name === elOfA.name;
+		return lengthsAreEqual && pathsAreEqual;
+	};
+
+	return candidate.files.every((elOfA) =>
+		searchee.files.some((elOfB) => cmp(elOfA, elOfB))
+	);
+}
```

Comment:
I can't tell the difference between this function and `compareFileTrees` above it. is it in progress?

Reply:
It compares only the filenames, not the paths. This differs when the files are nested for path but not for name.

Reply:
I think it's comparing paths per lines 94 and 95

Reply:
Yes, sorry. Pushing that now.